src/js/load-package.ts

```diff
@@ -165,7 +192,14 @@ class PackageManager {
     const loadedPackageData = new Set<InternalPackageData>();
     const messageCallback = options.messageCallback || console.log;
     const errorCallback = options.errorCallback || console.error;
-    if (names instanceof PyProxy) {
+
+    // originally, this condition was "names instanceof PyProxy",
+    // but it is changed to check names.toJs so that we can use type-only import for PyProxy and remove side effects.
+    // this change is required to run unit tests against this file, when global API or Module is not available.
+    // TODO: remove side effects from pyproxy.ts so that we can directly import PyProxy
```

Comment:
Huh, I wonder how hard this will be.

Reply:
But this replacement code isn't awful to be honest.

Reply:
I think we can fix it by cleaning up the pyproxy.ts code a bit, but it didn't work out easily the first few times I tried.